src/app/pages/LandingPage.svelte

```diff
@@ -92,9 +92,9 @@
 
   .masthead {
     /* These hacky variables are here */
-    --wave-height: 222px;
+    --wave-height: 333px;
     --masthead-space-top: 1rem;
-    --masthead-content-height: 13rem;
+    --masthead-content-height: 20rem;
```

Comment:
Out of curiosity, I assume that the images got too crowded on mobile with the previous height?

Reply:
This one is weird because if you try smaller values it creates a gap in the background:
![Screen Shot 2021-02-26 at 6 23 43 AM](https://user-images.githubusercontent.com/28884850/109311862-2e2b0500-77fb-11eb-91e9-c63dc154aa76.png)

`20rem` was just the right value to fill the gap, but let me know if you have a suggestion!

Reply:
Ah, makes sense
I played around with the CSS variables a bit, but I couldn't immediately come up with a viable alternative.
This works!